packages/now-static-build/src/frameworks.ts

```diff
@@ -413,7 +413,8 @@ const frameworkList: Framework[] = [
   {
     name: 'Jekyll',
     slug: 'jekyll',
-    buildCommand: 'jekyll build',
+    buildCommand: 'rm -f /usr/local/bundle/config && export BUNDLE_PATH=vendor/bundle && jekyll build',
```

Comment:
I have no idea how the filesystem is on Zeit, so I guessed based on the [official Docker Image](https://hub.docker.com/r/jekyll/jekyll) that there might me a configuration file here and I removed it since it has higher priority than the configuration I put there.

Maybe this can all be removed, but in order to do that someone with access to the filesystem should run `bundle config` to see how things are set up.